### PR TITLE
MRG: Fix circle [circle full] [ci skip]

### PR DESCRIPTION
**Pull request by Eric89GXL**
Circle has been failing (my bad) this should hopefully fix it

### CODE REVIEW

File: circle.yml [hunk 1]
```diff
@@ -112,7 +112,7 @@ dependencies:
     - echo BUILD="$(cat build.txt)"
     - ls -al ~/mne_data;
     - if [[ $(cat build.txt) == "html_dev" ]] || [[ $(cat build.txt) == "html_stable" ]]; then
-        python -c "import mne; mne.datasets._download_all_example_data()";
+        SUBJECTS_DIR=/home/ubuntu/mne_data/MNE-sample-data/subjects python -c "import mne; mne.datasets._download_all_example_data()";
```
Comment (Eric89GXL) [inline circle.yml]: This is how you set env vars for a particular call in bash. In this case we need SUBJECTS_DIR to be set, but really only for the call to the data downloading.

**Comment by agramfort** [comment]
LGTM but circle is not happy yet

**Comment by Eric89GXL** [comment]
Looking at the log it passes except fails to link docs, probably because scipy.org is down. So feel free to merge or we can wait until it's back up and I can restart

**Comment by agramfort** [comment]
ok thanks @Eric89GXL

**Pull request merged**